sfe.api/Controllers/ClientController.cs

```diff
@@ -16,6 +16,12 @@ public IEnumerable<Client> Get()
             return bll.ClientLogic.Get();
         }
 
+        // GET api/<controller>/5/5
+        public IEnumerable<Client> Get(int clientTypeId, int userId)
+        {
+            return bll.ClientLogic.Get(clientTypeId, userId);
```

Comment:
For BLL operation names use CRUD (create, read, update, delete), because those operations not necessary are going to be used by a rest.